internal/providers/docker/components/container/lifecycle.go

```diff
@@ -45,6 +46,15 @@ func (c *Container) create(ctx context.Context) error {
 		}
 	}
 
+	mounts := make([]mount.Mount, len(c.Spec.Volumes))
```

Comment:
same nitpicky feedback as a previous PR: don't need this temporary, can directly add mounts in to the `containerCfg` object. I prefer that style for a few reasons: 1) it's one fewer mistake to make, forgetting to assign the value 2) one line shorter for every case of having to build these things up 3) eliminates any clarity about whether or not the temporary is useful anywhere else.